build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id("java-library")
     id("maven-publish")
 
+    id("com.exactpro.th2.gradle.base") version "0.0.6"
```

Comment:
note: This is added automatically by `grpc` or `publish` plugin if they are applied to the root project